src/Templates/OrchardCore.ProjectTemplates/content/OrchardCore.Templates.Cms.Module/.template.config/dotnetcli.host.json

```diff
@@ -12,6 +12,6 @@
   },
   "usageExamples": [
     "--framework net8.0",
-    "--orchard-version 2.0.0"
+    "--orchard-version 2.1.0"
```

Comment:
Opening and building the solution on this branch in VS after cleaning the repo (i.e. no leftover build artifacts) causes such changes for me in each of them:

![{BD36F060-1205-46F3-BBED-E429ACED07E5}](https://github.com/user-attachments/assets/f60255ee-8eae-4e9a-954d-d9fe700095a1)

So, the version should be changed somewhere else (see https://github.com/search?q=repo%3AOrchardCMS%2FOrchardCore%201.8.3&type=code) and it documented in the release issue template.

Reply:
I have the same issue which is why I submitted this PR. I don't really know what are these files used for and what can we do to prevent them from being generated when we build

Reply:
But this PR doesn't fix the issue.

Reply:
If you have the fix, please fix it

Reply:
I don't have it.

Reply:
The solution is in this PR https://github.com/OrchardCMS/OrchardCore/pull/16733

I closed this PR